### DISCUSSION TITLE
TaskGroup default_args

### DISCUSSION THREAD

**Discussion by Pocoder** [apache/airflow#24523] [Q&A] [answered]
I want to use task groups as a reusable blocks in my dags and parametrise it through default_args conf. Here is simple example:
```python
from airflow.operators.python import get_current_context

@task
def return_greeting():
    return 'hello'

@task
def hello(some_value):
    greeting_prefix = get_current_context()['conf']['greeting']
    return f'{greeting_prefix} {some_value}')

@task_group
def simple_tg(some_value):
    result = hello(some_value)
    return result

@dag
def simple_dag():
    greeting = return_greeting()
    # Here I want to do smth like this
    res = simple_tg.override(default_args={'conf': {'greeting': greeting}})('Anna')
```
Does task_group decorated TaskGroup have some `override` like method for changing its params ([tasks can do it](https://airflow.apache.org/docs/apache-airflow/stable/tutorial_taskflow_api.html#reusing-a-decorated-task))? And can I pass XComArg (result of previous task) to default_args of task_group?

**Comment by potiuk** [comment, marked as answer]
No. But you could add it if you want - PRs are most welcome.
